nxcomp/Loop.cpp

```diff
@@ -9340,6 +9341,100 @@ int ParseCommandLineOptions(int argc, const char **argv)
 
           return -1;
         }
+        case 'd':
+        {
+          if ( argi+1 >= argc )
+          {
+            PrintUsageInfo(nextArg, 0);  
+            return -1;
+          }
+
+          int level = 0;
+	  level = strtol(argv[argi+1], NULL, 10);
```

Comment:
Another comment... you switched from using spaces to tabs here sometimes, while the rest of the file still uses spaces.